app/src/main/java/com/findmyrecycling/MainActivity.kt

```diff
@@ -35,15 +35,15 @@ fun RecycleSearch(name: String) {
     var location by remember{ mutableStateOf("")}
 
     Column {
-        OutlinedTextField(
+        TextField(
```

Comment:
Made changes to this based on what I found for functionality and looks. https://uxmovement.com/forms/why-you-shouldnt-use-solid-or-underlined-text-fields/

Reply:
Duplicates a previous review.  When that happens, first in wins.